config/foreman-proxy-content-answers.yaml

```diff
@@ -14,6 +14,7 @@ certs:
 foreman_proxy_content:
   proxy_pulp_isos_to_pulpcore: false
   proxy_pulp_yum_to_pulpcore: false
+  pulpcore_media_root: /var/lib/pulp
```

Comment:
This matches the default, so no need to include this.

Reply:
It's included to make it easy to tell new installs apart from upgrades.

Reply:
https://github.com/theforeman/foreman-installer/pull/543/files#diff-77cd90f81394cb4c12f94334eb5e1489R9